packages/app/src/systems/Core/components/Layout/Layout.tsx

```diff
@@ -45,22 +48,22 @@ type LayoutComponent = FC<LayoutProps> & {
 export const Layout: LayoutComponent = ({
   isPublic,
   isLoading,
+  isHome,
   title,
   children,
 }: LayoutProps) => {
-  const titleText = title ? `${title} | Fuel` : "Fuel";
+  const titleText = title ? `${title} | Fuel` : 'Fuel';
   return (
-    <ctx.Provider value={{ isLoading }}>
+    <ctx.Provider value={{ isLoading, isHome, title }}>
       <Helmet>
         <title>{titleText}</title>
       </Helmet>
       <Flex as="main" css={styles.root({ isPublic })}>
-        {isPublic ? (
-          <>{children}</>
-        ) : (
-          <Flex css={styles.wrapper}>{children}</Flex>
-        )}
+        {isPublic ? <>{children}</> : <Flex css={styles.wrapper}>{children}</Flex>}
```

Comment:
Can we keep the rule for didn't change these cases? It's more readable with the `()`